### PR TITLE
Remove Static Build

### LINKED ISSUE #35: Could Not Find Static Manifest
When a project is deployed to Heroku, the error

> Could not find static manifest at "/app/_build/prod/lib/web_delivery/priv/static/manifest.json". Run "mix phoenix.digest" after building your static files or remove the configuration from "config/prod.exs".

is generated. The Phoenix Booster Kit should not create any errors when it is deployed.

[Rollbar Error](https://rollbar.com/Spartan/booster-kit-phoenix/items/1/)

### PR DESCRIPTION
**Pull request by samcdavid**
Why
---

- The api does not have static assets and building static assets without
digesting them creates an error.

This change addresses the need by
---------------------------------

- Removing the static assets build config.

Related Issues
--------------

- Resolves #35

Next Steps
----------

- :shipit:

### CODE REVIEW

**Comment by mjs2600** [comment]
There's still some static code in `WebDelivery.Endpoint` and `./priv/static`. Also, you shouldn't need `Phoenix.Html`.

**Comment by samcdavid** [comment]
Where should we store `phoenix.js` so the react kit can get it? Is that even something that we need to worry about?

**Comment by endersstocker** [comment]
I wouldn’t include `phoenix.js` since it won’t be used on the Phoenix side and can be quickly acquired for the front end client. Fortunately the channel interop is relatively stable, so alignment between Phoenix version and `phoenix.js` versions shouldn’t be much of an issue.

**Comment by endersstocker** [comment]
P.S. It’s available as an npm package.

**Comment by samcdavid** [comment]
Thanks @endersstocker. Do you think `robots.txt` should remain since we don't want the api crawled by the google spider?

**Comment by endersstocker** [comment]
You can pull it out and create an issue to block crawlers. We could add a route or plug even that returns the necessary response since the contents of `robots.txt` won’t likely change.

**Comment by samcdavid** [comment]
@endersstocker 🔨 'ed it

File: apps/web_delivery/lib/web_delivery/endpoint.ex [hunk 1]
```diff
@@ -9,7 +9,7 @@ defmodule WebDelivery.Endpoint do
   # when deploying your static files in production.
   plug Plug.Static,
```
Comment (endersstocker) [inline apps/web_delivery/lib/web_delivery/endpoint.ex]: You can _unplug_ this now.

Comment (samcdavid) [inline apps/web_delivery/lib/web_delivery/endpoint.ex]: @endersstocker got 'em

**Pull request merged**